Apps/GatewayApi/src/Models/UserProfileModel.cs

```diff
@@ -105,28 +105,5 @@ public UserProfileModel(IDictionary<string, UserPreferenceModel> preferences, IL
         /// Gets the user preference.
         /// </summary>
         public IDictionary<string, UserPreferenceModel> Preferences { get; } = new Dictionary<string, UserPreferenceModel>();
-
-        /// <summary>
-        /// Constructs a UserProfile model from a UserProfile database model.
-        /// </summary>
-        /// <param name="model">The user profile database model.</param>
-        /// <param name="latestTermsOfServiceId">The GUID corresponding to the most recent terms of service.</param>
-        /// <returns>The user profile model.</returns>
-        public static UserProfileModel CreateFromDbModel(UserProfile model, Guid? latestTermsOfServiceId)
-        {
-            return new UserProfileModel()
-            {
-                HdId = model.HdId,
-                TermsOfServiceId = model.TermsOfServiceId,
-                Email = model.Email,
-                IsEmailVerified = !string.IsNullOrEmpty(model.Email),
-                SMSNumber = model.SMSNumber,
-                IsSMSNumberVerified = !string.IsNullOrEmpty(model.SMSNumber),
-                LastLoginDateTime = model.LastLoginDateTime,
-                ClosedDateTime = model.ClosedDateTime,
-                AcceptedTermsOfService = model.TermsOfServiceId != Guid.Empty,
-                HasTermsOfServiceUpdated = model.TermsOfServiceId != latestTermsOfServiceId,
-            };
```

Comment:
It looks like UserProfileMapUtils only handles the calculation of the HasTermsOfServiceUpdated property and not AcceptedTermsOfService, IsEmailVerified, or IsSMSNumberVerified.

Reply:
The other three are handled directly by the UserProfileProfile.